networking/aws-load-balancer-operator.adoc

```diff
@@ -24,4 +24,5 @@ endif::openshift-rosa[]
 clusters.
 
 include::modules/aws-installing-an-aws-load-balancer-operator.adoc[leveloffset=+1]
+include::modules/nw-aws-load-balancer-with-outposts.adoc[leveloffset=+1]
```

Comment:
This is still in the wrong assembly looks like - you want it in the place you mention in the module use statement above (`networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc`)

Reply:
TY